presto-hive/src/main/java/io/prestosql/plugin/hive/HivePageSource.java

```diff
@@ -361,6 +364,15 @@ public ConnectorPageSource getPageSource()
         if (toType instanceof VarcharType && (fromHiveType.equals(HIVE_BYTE) || fromHiveType.equals(HIVE_SHORT) || fromHiveType.equals(HIVE_INT) || fromHiveType.equals(HIVE_LONG))) {
             return new IntegerNumberToVarcharCoercer<>(fromType, (VarcharType) toType);
         }
+        if (toType instanceof VarcharType && fromHiveType.equals(HIVE_FLOAT)) {
+            return new FloatToVarcharCoercer<>(fromType, (VarcharType) toType);
```

Comment:
What about moving all coercers with `toType == VarcharType` into a single class similar to `DecimalCoercers`? Or are separate classes more readable?